docs/Stream-API.md

```diff
@@ -0,0 +1,473 @@
+---
+title: Stream API Overview
+layout: documentation
+documentation: true
+---
+
+* [Concepts](#concepts)
+    * [Stream Builder] (#streambuilder)
+    * [Value mapper] (#valuemapper)
+* [Stream APIs](#streamapis)
+    * [Basic transformations] (#basictransformations)
```

Comment:
Same as the above comment to remove space between [] and () for generating links till aggregations section.